launch/personal_food_computer_v2.launch

```diff
@@ -1,7 +1,9 @@
 <launch>
+  <arg name="environment_1" value="environments/environment_1" />
+
```

Comment:
That's there since args are like configurable parameters for the launchfile, and it's a fairly common launchfile thing to do. Think of them like a file-wide CONST_VARIABLE.